frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseRebaseBranchOntoParentAction.java

```diff
@@ -47,7 +47,7 @@ public abstract class BaseRebaseBranchOntoParentAction extends BaseGitMacheteRep
   private static final String NL = System.lineSeparator();
 
   @Override
-  public IEnhancedLambdaLogger log() {
+  public LambdaLogger log() {
```

Comment:
At least this is now wrapper over slf4j, not a wrapper over wrapper over slf4j 😅